.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]
+end_of_line = lf
```

Comment:
So you, check the native eol into git? So files written by Windows use crlf and unix lf?

> core.autocrlf = false This is the default. The result of using false is that Git doesn’t ever mess with line endings on your file. You can check in files with LF or CRLF or CR or some random mix of those three and Git does not care.
http://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/

will delete that line on the next rebase